docs/notifications/concepts/notification.md

```diff
@@ -0,0 +1,42 @@
+Notification allows you to receive alerts on a preferred channel when a particular set of events occur.
+
+![Create notification form](../../images/create-notification-form.png)
```

Comment:
I don't think it is necessary to add screenshots as then we will need to update them